### PR TITLE
5006 Init Project

### PR DESCRIPTION
**Pull request by manhbn30594**

### CODE REVIEW

**Comment by manhbn30594** [comment]
https://edu-redmine.framgia.vn/issues/5006

**Comment by manhbn30594** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+ruby "2.5.1"
+gem "rails", "~> 5.2.2"
+gem "sqlite3"
+gem "puma", "~> 3.11"
+gem "sass-rails", "~> 5.0"
+gem "uglifier", ">= 1.3.0"
+gem "coffee-rails", "~> 4.2"
```
Comment (mdmahmudurrahman) [inline Gemfile]: Don't need this gem. Please remove. :D

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>ROe13HealthySale</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (mdmahmudurrahman) [inline app/views/layouts/application.html.erb]: 1. Please use Double quote in ruby
2. Remove extra white space between parameter

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
```
Comment (mdmahmudurrahman) [inline config/database.yml]: 1. Please copy this file and rename it like as database.yml.example 
2. Add database.yml in the .gitignore

File: config/locales/en.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
+# To learn more, please read the Rails Internationalization guide
+# available at http://guides.rubyonrails.org/i18n.html.
+
```
Comment (mdmahmudurrahman) [inline config/locales/en.yml]: You can remove all of the comments for the locale

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by manhbn30594** [comment]
ready

File: config/database.yml (hunk 3 above)
Comment (mdmahmudurrahman) [reply config/database.yml]: @manhbn30594 Em, remove this file from git commit. 
You can try the command
```ruby
git rm file_name
```

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by manhbn30594** [comment]
ready

**Pull request merged**